scripts/build_food_db.py

```
@@ -75,21 +75,21 @@ def load_source_data(self) -> tuple[List[Dict], List[Dict]]:
-            adapter = OFFAdapter(str(self.off_chunks_dir))
+            adapter = OFFAdapter(str(self.off_chunks_dir))  # type: ignore[assignment]
         else:
             print("  📊 Loading OFF from single file")
-            adapter = OFFAdapter()
+            adapter = OFFAdapter()  # type: ignore[assignment]
 
         try:
-            off_data = list(adapter.normalize())
+            off_data = list(adapter.normalize())  # type: ignore[arg-type]
             print(f"  ✅ OFF: {len(off_data)} records")
         except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)